src/texturebrush.c

```diff
@@ -770,7 +785,7 @@ GdipCreateTextureIAI (GpImage *image, GpImageAttributes *imageAttributes, int x,
 {
 	/* FIXME MonoTODO: Make use of ImageAttributes parameter when
 	 * ImageAttributes is implemented */
```

Comment:
Is this FIXME still relevant?

Reply:
Yeah we still need to apply the image attributes to the image (e.g. recoloring)